libs/vectorise/tests/gtest/avx_tests.cpp

```diff
@@ -99,10 +100,12 @@ using MyTypes = ::testing::Types<
     fetch::vectorise::VectorRegister<double, 128>, fetch::vectorise::VectorRegister<double, 256>>;
 
 using MyFPTypes =
-    ::testing::Types<fetch::vectorise::VectorRegister<float, 256>,
+    ::testing::Types</*fetch::vectorise::VectorRegister<float, 256>,*/
```

Comment:
these types became unsupported?

Reply:
no, I removed them completely, the particular test is for checking the NaN/Inf/etc states work with SIMD FixedPoint, we already know that these work with floats/doubles already.